### PR TITLE
Feat/resize textarea

### PR DESCRIPTION
**Pull request by andresz1**
## Disable textarea resize

**TASK**: #1099

### Description, Motivation and Context
This PR:
- Add `isResizable` prop to textarea
- Missing `data-spark-component` attributes

### Types of changes
- [ ] 🛠️ Tool
- [ ] 🪲 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles

### CODE REVIEW

File: packages/components/textarea/src/Textarea.tsx [hunk 1]
```diff
@@ -2,12 +2,26 @@ import { Input } from '@spark-ui/input'
...
 const Root = forwardRef<HTMLTextAreaElement, PropsWithChildren<TextareaProps>>(
-  ({ className, disabled, rows = 1, ...others }, ref) => {
+  ({ className, disabled, rows = 1, isResizeDisabled = false, ...others }, ref) => {
     return (
-      <Input className={cx(className, 'py-[var(--sz-10)] resize-y')} disabled={disabled} asChild>
+      <Input
+        className={cx(
+          className,
+          'py-[var(--sz-10)]',
+          isResizeDisabled ? 'resize-none' : 'resize-y'
```
Comment (andresz1) [inline packages/components/textarea/src/Textarea.tsx:19]: Not `100%` sure about this one.
- Is the name of prop ok ?
- Should we let them control this by using `className` or is it better a `prop` like this ?

Comment (soykje) [reply packages/components/textarea/src/Textarea.tsx:19]: :wave: I would vote for leaving the possibility to developper to toggle resize on textarea through custom styles (`className`), but I guess this also could be discussed with design team? Elseway I'd prefer maybe a simpler prop name such as `isResizable` wdyt?

Comment (andresz1) [reply packages/components/textarea/src/Textarea.tsx:19]: I like `isResizable` better. However the downside is that the default value should be `true`. That's the reason why I chose `isResizeDisabled`. Perhaps we can discuss this during the day

Comment (soykje) [reply packages/components/textarea/src/Textarea.tsx:19]: Hmmm got it, I didn't understood that we wanted to keep the resize feat. by default :sweat_smile: Well, maybe we should discuss first with design team, and then see together :+1:

**Review by Powerplex — APPROVED** [review]

File: packages/components/textarea/src/Textarea.tsx (hunk 1 above)
Comment (acd02) [reply packages/components/textarea/src/Textarea.tsx:19]: I vote for `isResizable` as well 👍

Comment (andresin87) [reply packages/components/textarea/src/Textarea.tsx:19]: and if we call it

`resizable`: boolean or vertical/horizontal: default true

**Review by andresin87 — APPROVED** [review]

**Review by soykje — APPROVED** [review]

**Review by turolopezsanabria — APPROVED** [review]
👏🏼

**Pull request merged**